### PR TITLE
Updated Preview UIs and Changes to Painscale

### LINKED ISSUE #7290: Log update- vitals
**Is your feature request related to a problem? Please describe.**
In Log updates (Critical round), Vitals section, there is a section to mark details of pain.

Against a region, user can mark scale of pain from 0-5. 

Screenshot 2024-02-27 at 3 35 23 PM

Here the text "Low" , "Normal", "High" comes written next to the number.

**Describe the solution you'd like**

It is not useful to tag pain as "Normal"

Only retain "Low" only at 0 and "High" at 5.

- [ ] remove the appearance of "normal" against 1-4 painscale

### LINKED ISSUE #7376: UI Enhancement of the Pressure Sore and Pain Scale Preview Mode
**Describe the bug**

currently, the preview mode of the pain scale and the pressure sore look like an input field, which creates a confusion for users

**To Reproduce**

1. Go to the patient consultation page
2. Click on the vital section in the nav tab
3. scroll down to the bottom 
4. click on any body part 
5. see the error

Similarly, Go to pressure sore in the nav tab and click on any body part to see the error

**Expected behavior**

- [ ] We should create a new UI for pain scale preview mode and pressure sore preview mode, which is more user-friendly.

**Note: the mockup UI design must be approved by the core team in the EOD call or the care_general channel before coding the solution.**

**Screenshots**

**Pain Scale Preview Mode**

image

**Pressure Sore Preview Mode**

image

### LINKED ISSUE #7378: Pain Scale (0-10)
**Is your feature request related to a problem? Please describe.**
https://care.coronasafe.in/facility/0c95c7f0-e1d2-4aff-83fa-933cef60d3a8/patient/7e8fc20f-02da-47f7-8c33-f2d8c7e52dd8/consultation/cb44a173-6fe2-452f-b2d1-6c6e2695411e/daily_rounds/8e4d0791-30d4-4e5d-8ca4-f6e27abb6418/update

Currently the pain scale in CARE is from 0-5.

Change it to 0-10 to allow user to input a more nuanced measure of pain experienced by the patient.

### PR DESCRIPTION
**Pull request by shivankacker**
Dependent on https://github.com/coronasafe/care/pull/1967

## Proposed Changes

- Fixes #7376
- Fixes #7378
- Fixes #7290
- Changes UI for Preview Modes in Pressure Sore and Pain Scale editor in patient consultation
image
image
- Changed Painscale range from 0-5 to 0-10

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [x] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Request for Peer Reviews
- [x] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 
image

- [ ] when i scroll the page, and keep exactly in the position as in the screenshot, then click on the right chest, the pop-up overlaps with the badges.
- [ ] Keep the edges of the card as a curve, as we have curve border across the platform

**Comment by shivankacker** [comment]
@nihal467 
the popup does that when it does not have space to appear below. Check this screenshot. I don't think this should have any UX problems. Other fixes are done.
image

**Comment by nihal467** [comment]
@skks1212 clear the merge conflict

**Comment by nihal467** [comment]
@skks1212 fix the merge conflict

**Comment by nihal467** [comment]
@skks1212 can you clear the merge conflict

**Comment by nihal467** [comment]
LGTM

**Review by rithviknishad — APPROVED** [review]

File: src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreInputModal.res [hunk 1]
```diff
@@ -54,11 +55,29 @@ let make = (
...
+
         hideModal(event)
+
+
+
       }
+
+
+
     }
```
Comment (khavinshankar) [inline src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureSoreInputModal.res]: @skks1212 is the spacing intensional

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**